### PR TITLE
CI: Submit the Application Payload to Microsoft store into CI

### PR DESCRIPTION
**Pull request by starnight**
Bring the actions to submit Endless Key to Microsoft store
automatically when release.

https://phabricator.endlessm.com/T33436

### CODE REVIEW

**Review by danigm — APPROVED** [review]
Looks good to me

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -157,3 +157,62 @@ jobs:
...
+            -Verbose
+
+          $submissionId, $submissionUrl = Update-ApplicationSubmission `
+            -AppId $APP_ID `
+            -SubmissionDataPath "$SubmissionRoot/endlesskey.json" `
+            -PackagePath "$SubmissionRoot/endlesskey.zip" `
+            -ReplacePackages `
+            -AutoCommit `
+            -Force `
+            -Verbose
```
Comment (jprvita) [inline .github/workflows/main.yml:222]: I think we should add `-UpdateListings` here to also update the screenshots -- rationale in https://phabricator.endlessm.com/T33436#940050

**Review by jprvita — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (starnight) [reply .github/workflows/main.yml:222]: Updated & place some discussion in https://phabricator.endlessm.com/T33436#940115

Comment (jprvita) [reply .github/workflows/main.yml:222]: Further discussion on the ticket.

**Review by jprvita — CHANGES_REQUESTED** [review]
Further discussion on https://phabricator.endlessm.com/T33436#940316

File: .github/workflows/main.yml (hunk 1 above)
Comment (starnight) [reply .github/workflows/main.yml:222]: Updated!

**Review by jprvita — APPROVED** [review]

**Pull request merged**